lib/types/documentdb.d.ts

```diff
@@ -0,0 +1,10 @@
+import * as documentdb from "documentdb";
+
+declare module "documentdb" {
+  // [#153802058] _ts is a number but is defined as string.
+  // TODO: get rid of this file when this pull request gets merged:
+  // https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22291#issuecomment-352883251
+  export interface RetrievedDocument extends NewDocument, AbstractMeta {
```

Comment:
is this extending or replacing the definition of `RetrievedDocument` ?

Reply:
this overrides the type of _ts in RetrievedDocument (inherited from AbstractMeta)

Reply:
I see 👍 

and I guess the `declare module "documentdb" {}` statement, actually extends the module declaration from DefinitelyTyped? (not intuitive at all, it looks like an override)

Reply:
that's called https://www.typescriptlang.org/docs/handbook/declaration-merging.html

Reply:
brrr